OneCommunityGlobal/HighestGoodNetworkApp
# Yixiao_form_unittest

# Description
Create the unit test for the Form component to confirm whether it working as intended.

## Main changes explained:
- Create file Form.test.js to do the unit test.
- Update file Form 


## How to test:
1. check into current branch
2. do npm install to run this PR locally
3. run npm  test src/components/common/Form/`__tests__`/Form.test.js

## Screenshots or videos of changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/49748716/4d6d29d7-f948-4eca-b7ba-c6405b50782b)
